src/helpers/stake/magicGlp/getAdditionalInfo.ts

```diff
@@ -0,0 +1,20 @@
+import { utils } from "ethers";
+
+export const getAdditionalInfo = async (contracts: any, config: any) => {
+  const { harvestorContract, chainLinkContract } = contracts;
+  const { rewardToken, leverageInfo } = config.additionalInfo;
+
+  const multicallArr = [
+    harvestorContract.feePercentBips(),
+    chainLinkContract.latestAnswer(),
+  ];
+
+  const [feePercentBips, rewardTokenPrice] = await Promise.all(multicallArr);
+
+  return {
+    feePercent: feePercentBips / 10000,
```

Comment:
have a BIPS constant somewhere set to 10_000

Reply:
added